test/plots/autoplot.js

```diff
@@ -210,5 +210,5 @@ export async function autoDotFacet2() {
     fx: "island",
     fy: "species",
     color: "sex"
-  }).plot();
+  }).plot({marginRight: 70, x: {ticks: 5}});
```

Comment:
This is another unrelated change (and also we should be using facet.marginRight, and I think it’s good as previously stated to have the Plot.auto tests be as short has possible since the focus is on speed and not looking good).